src/styles/main.scss

```diff
@@ -1,3 +1,16 @@
-body {
+@import "src/styles/components/header.scss";
+@import "src/styles/components/nav.scss";
+@import "src/styles/components/variables.scss";
```

Comment:
The 'variables.scss' file should be imported first because it may contain variables that are used in other scss files.